_posts/2020-05-25-flink-on-zeppelin-part1.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Flink on Zeppelin Notebooks for Interactive Data Analysis - Part 1"
+date:   2020-05-25T08:00:00.000Z
```

Comment:
Date needs to be updated before merging.

Reply:
I updated the date as next Monday, does it make sense to publish it next Monday ? @morsapaes

Reply:
Thanks, Jeff! I will make sure this is published tomorrow.

Reply:
Thanks @morsapaes